### PR TITLE
fix some caps

### PR DESCRIPTION
**Pull request by erikbor**
Hi @phuonghuynh, I turned some lower case into upper case.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -8,7 +8,7 @@ output "execution_resources" {
     "${module.gateway_authorize.execution_resource}",
     "${module.gateway_revoke.execution_resource}"
   ]
-  description = "Excution resouces arn"
+  description = "Excution resouces ARN"
```
Comment (ronaldtse) [inline outputs.tf:11]: Typo here too

File: outputs.tf [hunk 2]
```diff
@@ -23,5 +23,5 @@ output "events" {
   value       = [
     "${aws_cloudwatch_event_rule.clear.arn}"
   ]
-  description = "Event arns"
+  description = "Event ARNs"
```
Comment (ronaldtse) [inline outputs.tf:26]: Should be `ARNs`

**Comment by ronaldtse** [comment]
@erikbor did you see my comments? Thanks

**Comment by erikbor** [comment]
Yes fixed in [#11](https://github.com/riboseinc/terraform-aws-iam-authenticating-group/pull/11l)

**Comment by ronaldtse** [comment]
Thanks @erikbor !

**Pull request merged**